ext/spl/spl_fixedarray.c

```diff
@@ -1035,7 +1035,7 @@ ZEND_BEGIN_ARG_INFO(arginfo_fixedarray_setSize, 0)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_fixedarray_fromArray, 0, 0, 1)
-	ZEND_ARG_INFO(0, data)
+	ZEND_ARG_INFO(0, array)
```

Comment:
Should update https://github.com/php/php-src/blob/0fe767d559cb52f38433e0e1c15c9f299058cca8/ext/spl/spl_fixedarray.c#L650.